### ISSUE TITLE
Wine 4 does not seem to work (Build error win64 Error: rcedit.exe failed  Library not loaded: /usr/local/Cellar/wine/4.0/lib/libwine.1.dylib)

### ISSUE THREAD

**Issue by LabDeve** [nwjs-community/nw-builder#513]
While building on osx, it throw this error:
`Build error win64 Error: rcedit.exe failed  Library not loaded: /usr/local/Cellar/wine/4.0/lib/libwine.1.dylib`

I resolved this issue by downgrading to wine 3.0.2

For anyone else having this issue:
  - Download specific version from `https://homebrew.bintray.com/bottles/wine-3.0.2.sierra.bottle.tar.gz`
  - Unzip and copy the content here `/usr/local/cellar/`
  - Run: `brew switch wine 3.0.2`

Or run: `wget -qO- https://homebrew.bintray.com/bottles/wine-3.0.2.sierra.bottle.tar.gz | bsdtar -xvf- -C /usr/local/cellar/; brew switch wine 3.0.2`

**Comment by flotpg** [comment]
Hey Team, same here for me with wine 4.0 - Yesterday I did a brew upgrade (I noticed winetricks had an issue creating/overwriting an existing symlink:
`brew upgrade
==> Upgrading 20 outdated packages:
aircrack-ng 1.4 -> 1.5.2, pyenv 1.2.8 -> 1.2.9, wget 1.19.5 -> 1.20.1_4, fdk-aac 0.1.6 -> 2.0.0, libidn2 2.0.5 -> 2.1.1a, azure-cli 2.0.57 -> 2.0.60, cmake 3.12.3 -> 3.14.0, python 3.7.2_1 -> 3.7.2_2, fio 3.8 -> 3.12, wine 3.0.3 -> 4.0, sqlite 3.26.0_1 -> 3.27.1, fdk-aac-encoder 1.0.0 -> 1.0.0_1, htop 2.2.0 -> 2.2.0_1, speedtest-cli 2.0.2 -> 2.1.1, lastpass-cli 1.3.1 -> 1.3.1_1, inetutils 1.9.4_1 -> 1.9.4_2, openssl@1.1 1.1.1 -> 1.1.1b, pcre 8.42 -> 8.43, winetricks 20181203 -> 20190310, ncdu 1.13 -> 1.14

brew upgrade
==> Upgrading 20 outdated packages:
aircrack-ng 1.4 -> 1.5.2, pyenv 1.2.8 -> 1.2.9, wget 1.19.5 -> 1.20.1_4, fdk-aac 0.1.6 -> 2.0.0, libidn2 2.0.5 -> 2.1.1a, azure-cli 2.0.57 -> 2.0.60, cmake 3.12.3 -> 3.14.0, python 3.7.2_1 -> 3.7.2_2, fio 3.8 -> 3.12, wine 3.0.3 -> 4.0, sqlite 3.26.0_1 -> 3.27.1, fdk-aac-encoder 1.0.0 -> 1.0.0_1, htop 2.2.0 -> 2.2.0_1, speedtest-cli 2.0.2 -> 2.1.1, lastpass-cli 1.3.1 -> 1.3.1_1, inetutils 1.9.4_1 -> 1.9.4_2, openssl@1.1 1.1.1 -> 1.1.1b, pcre 8.42 -> 8.43, winetricks 20181203 -> 20190310, ncdu 1.13 -> 1.14
==> Upgrading openssl@1.1
==> Downloading https://homebrew.bintray.com/bottles/openssl@1.1-1.1.1b.mojave.bottle.tar.gz
######################################################################## 100.0%
==> Pouring openssl@1.1-1.1.1b.mojave.bottle.tar.gz
==> Caveats
A CA file has been bootstrapped using certificates from the system
keychain. To add additional certificates, place .pem files in
  /usr/local/etc/openssl@1.1/certs

and run
  /usr/local/opt/openssl@1.1/bin/c_rehash

openssl@1.1 is keg-only, which means it was not symlinked into /usr/local,
because openssl/libressl is provided by macOS so don't link an incompatible version.

If you need to have openssl@1.1 first in your PATH run:
  echo 'export PATH="/usr/local/opt/openssl@1.1/bin:$PATH"' >> ~/.bash_profile

For compilers to find openssl@1.1 you may need to set:
  export LDFLAGS="-L/usr/local/opt/openssl@1.1/lib"
  export CPPFLAGS="-I/usr/local/opt/openssl@1.1/include"

For pkg-config to find openssl@1.1 you may need to set:
  export PKG_CONFIG_PATH="/usr/local/opt/openssl@1.1/lib/pkgconfig"

==> Summary
🍺  /usr/local/Cellar/openssl@1.1/1.1.1b: 7,957 files, 17.9MB
Removing: /usr/local/Cellar/openssl@1.1/1.1.0f... (6,421 files, 15.5MB)
Removing: /usr/local/Cellar/openssl@1.1/1.1.0g... (6,585 files, 15.6MB)
Removing: /usr/local/Cellar/openssl@1.1/1.1.0g_1... (6,585 files, 15.6MB)
Removing: /usr/local/Cellar/openssl@1.1/1.1.0h... (6,587 files, 15.6MB)
Removing: /usr/local/Cellar/openssl@1.1/1.1.1... (7,821 files, 17.9MB)
==> Upgrading sqlite
==> Downloading https://homebrew.bintray.com/bottles/sqlite-3.27.1.mojave.bottle.tar.gz
######################################################################## 100.0%
==> Pouring sqlite-3.27.1.mojave.bottle.tar.gz
==> Caveats
sqlite is keg-only, which means it was not symlinked into /usr/local,
because macOS provides an older sqlite3.

If you need to have sqlite first in your PATH run:
  echo 'export PATH="/usr/local/opt/sqlite/bin:$PATH"' >> ~/.bash_profile

For compilers to find sqlite you may need to set:
  export LDFLAGS="-L/usr/local/opt/sqlite/lib"
  export CPPFLAGS="-I/usr/local/opt/sqlite/include"

For pkg-config to find sqlite you may need to set:
  export PKG_CONFIG_PATH="/usr/local/opt/sqlite/lib/pkgconfig"

==> Summary
🍺  /usr/local/Cellar/sqlite/3.27.1: 11 files, 3.7MB
Removing: /usr/local/Cellar/sqlite/3.26.0_1... (11 files, 3.7MB)
==> Upgrading wget
==> Installing dependencies for wget: libidn2
==> Installing wget dependency: libidn2
==> Downloading https://homebrew.bintray.com/bottles/libidn2-2.1.1a.mojave.bottle.tar.gz
######################################################################## 100.0%
==> Pouring libidn2-2.1.1a.mojave.bottle.tar.gz
🍺  /usr/local/Cellar/libidn2/2.1.1a: 69 files, 691.8KB
==> Installing wget
==> Downloading https://homebrew.bintray.com/bottles/wget-1.20.1_4.mojave.bottle.tar.gz
######################################################################## 100.0%
==> Pouring wget-1.20.1_4.mojave.bottle.tar.gz
🍺  /usr/local/Cellar/wget/1.20.1_4: 49 files, 3.9MB
Removing: /usr/local/Cellar/wget/1.19.1_1... (11 files, 1.6MB)
Removing: /usr/local/Cellar/wget/1.19.2... (11 files, 1.7MB)
Removing: /usr/local/Cellar/wget/1.19.4_1... (50 files, 3.7MB)
Removing: /usr/local/Cellar/wget/1.19.5... (50 files, 3.7MB)
==> Upgrading fdk-aac
==> Downloading https://homebrew.bintray.com/bottles/fdk-aac-2.0.0.mojave.bottle.tar.gz
######################################################################## 100.0%
==> Pouring fdk-aac-2.0.0.mojave.bottle.tar.gz
🍺  /usr/local/Cellar/fdk-aac/2.0.0: 15 files, 3MB
Removing: /usr/local/Cellar/fdk-aac/0.1.5... (14 files, 1.8MB)
Removing: /usr/local/Cellar/fdk-aac/0.1.6... (14 files, 1.8MB)
Removing: /usr/local/Cellar/libidn2/2.0.4... (46 files, 580.6KB)
Removing: /usr/local/Cellar/libidn2/2.0.5... (68 files, 668.6KB)
==> Upgrading azure-cli
==> Installing dependencies for azure-cli: python
==> Installing azure-cli dependency: python
Warning: Building python from source:
  The bottle needs the Apple Command Line Tools to be installed.
  You can install them, if desired, with:
    xcode-select --install

==> Downloading https://www.python.org/ftp/python/3.7.2/Python-3.7.2.tgz
Already downloaded: /Users/flo/Library/Caches/Homebrew/downloads/c0a14628b5078ca696f1e18227df67d9c5fbe04462c041d9252709c120ea3e1a--Python-3.7.2.tgz
==> ./configure --prefix=/usr/local/Cellar/python/3.7.2_2 --enable-ipv6 --datarootdir=/usr/local/Cellar/python/3.7.2_2
==> make
==> make install PYTHONAPPSDIR=/usr/local/Cellar/python/3.7.2_2
==> make frameworkinstallextras PYTHONAPPSDIR=/usr/local/Cellar/python/3.7.2_2/share/python
==> Downloading https://files.pythonhosted.org/packages/c2/f7/c7b501b783e5a74cf1768bc174ee4fb0a8a6ee5af6afa92274ff9647
######################################################################## 100.0%
==> Downloading https://files.pythonhosted.org/packages/4c/4d/88bc9413da11702cbbace3ccc51350ae099bb351febae8acc85fec34
######################################################################## 100.0%
==> Downloading https://files.pythonhosted.org/packages/d9/7d/86df15e317027f6e87aa68ea854abf8437e796b4c0fadd3ae5ee67b7
######################################################################## 100.0%
==> /usr/local/Cellar/python/3.7.2_2/bin/python3 -s setup.py --no-user-cfg install --force --verbose --install-scripts
==> /usr/local/Cellar/python/3.7.2_2/bin/python3 -s setup.py --no-user-cfg install --force --verbose --install-scripts
==> /usr/local/Cellar/python/3.7.2_2/bin/python3 -s setup.py --no-user-cfg install --force --verbose --install-scripts
==> Caveats
Python has been installed as
  /usr/local/bin/python3

Unversioned symlinks `python`, `python-config`, `pip` etc. pointing to
`python3`, `python3-config`, `pip3` etc., respectively, have been installed into
  /usr/local/opt/python/libexec/bin

If you need Homebrew's Python 2.7 run
  brew install python@2

You can install Python packages with
  pip3 install <package>
They will install into the site-package directory
  /usr/local/lib/python3.7/site-packages

See: https://docs.brew.sh/Homebrew-and-Python
==> Summary
🍺  /usr/local/Cellar/python/3.7.2_2: 8,464 files, 118.4MB, built in 3 minutes 39 seconds
==> Installing azure-cli
==> Downloading https://homebrew.bintray.com/bottles/azure-cli-2.0.60.mojave.bottle.tar.gz
######################################################################## 100.0%
==> Pouring azure-cli-2.0.60.mojave.bottle.tar.gz
==> Caveats
Bash completion has been installed to:
  /usr/local/etc/bash_completion.d
==> Summary
🍺  /usr/local/Cellar/azure-cli/2.0.60: 24,831 files, 106.5MB
Removing: /usr/local/Cellar/azure-cli/2.0.57... (22,383 files, 100.5MB)
==> Upgrading cmake
==> Downloading https://homebrew.bintray.com/bottles/cmake-3.14.0.mojave.bottle.tar.gz
######################################################################## 100.0%
==> Pouring cmake-3.14.0.mojave.bottle.tar.gz
==> Caveats
Emacs Lisp files have been installed to:
  /usr/local/share/emacs/site-lisp/cmake
==> Summary
🍺  /usr/local/Cellar/cmake/3.14.0: 5,680 files, 53.2MB
Removing: /usr/local/Cellar/cmake/3.11.4... (2,363 files, 32.7MB)
Removing: /usr/local/Cellar/cmake/3.12.3... (2,417 files, 33.3MB)
Removing: /usr/local/Cellar/python/3.7.2_1... (8,437 files, 118MB)
==> Upgrading fio
==> Downloading https://homebrew.bintray.com/bottles/fio-3.12.mojave.bottle.tar.gz
######################################################################## 100.0%
==> Pouring fio-3.12.mojave.bottle.tar.gz
🍺  /usr/local/Cellar/fio/3.12: 21 files, 1.4MB
Removing: /usr/local/Cellar/fio/3.0... (21 files, 1.4MB)
Removing: /usr/local/Cellar/fio/3.1... (21 files, 1.4MB)
Removing: /usr/local/Cellar/fio/3.2... (21 files, 1.4MB)
Removing: /usr/local/Cellar/fio/3.4... (21 files, 1.4MB)
Removing: /usr/local/Cellar/fio/3.5... (21 files, 1.4MB)
Removing: /usr/local/Cellar/fio/3.7... (21 files, 1.4MB)
Removing: /usr/local/Cellar/fio/3.8... (21 files, 1.4MB)
==> Upgrading wine
==> Downloading https://homebrew.bintray.com/bottles/wine-4.0.sierra.bottle.tar.gz
######################################################################## 100.0%
==> Pouring wine-4.0.sierra.bottle.tar.gz
==> Regenerating font cache, this may take a while
==> /usr/local/Cellar/wine/4.0/libexec/bin/fc-cache -frv
==> Caveats
You may also want winetricks:
  brew install winetricks
==> Summary
🍺  /usr/local/Cellar/wine/4.0: 8,989 files, 656.4MB
Removing: /usr/local/Cellar/wine/3.0.2... (8,656 files, 609.5MB)
Removing: /usr/local/Cellar/wine/3.0.3... (8,701 files, 608.7MB)
==> Upgrading fdk-aac-encoder
==> Downloading https://homebrew.bintray.com/bottles/fdk-aac-encoder-1.0.0_1.mojave.bottle.tar.gz
######################################################################## 100.0%
==> Pouring fdk-aac-encoder-1.0.0_1.mojave.bottle.tar.gz
🍺  /usr/local/Cellar/fdk-aac-encoder/1.0.0_1: 9 files, 107.8KB
Removing: /usr/local/Cellar/fdk-aac-encoder/0.6.3... (8 files, 101.2KB)
Removing: /usr/local/Cellar/fdk-aac-encoder/1.0.0... (9 files, 107.8KB)
==> Upgrading htop
==> Installing dependencies for htop: ncurses
==> Installing htop dependency: ncurses
==> Downloading https://homebrew.bintray.com/bottles/ncurses-6.1.mojave.bottle.tar.gz
######################################################################## 100.0%
==> Pouring ncurses-6.1.mojave.bottle.tar.gz
==> Caveats
ncurses is keg-only, which means it was not symlinked into /usr/local,
because macOS already provides this software and installing another version in
parallel can cause all kinds of trouble.

If you need to have ncurses first in your PATH run:
  echo 'export PATH="/usr/local/opt/ncurses/bin:$PATH"' >> ~/.bash_profile

For compilers to find ncurses you may need to set:
  export LDFLAGS="-L/usr/local/opt/ncurses/lib"
  export CPPFLAGS="-I/usr/local/opt/ncurses/include"

For pkg-config to find ncurses you may need to set:
  export PKG_CONFIG_PATH="/usr/local/opt/ncurses/lib/pkgconfig"

==> Summary
🍺  /usr/local/Cellar/ncurses/6.1: 3,869 files, 8.3MB
==> Installing htop
==> Downloading https://homebrew.bintray.com/bottles/htop-2.2.0_1.mojave.bottle.tar.gz
######################################################################## 100.0%
==> Pouring htop-2.2.0_1.mojave.bottle.tar.gz
==> Caveats
htop requires root privileges to correctly display all running processes,
so you will need to run `sudo htop`.
You should be certain that you trust any software you grant root privileges.
==> Summary
🍺  /usr/local/Cellar/htop/2.2.0_1: 11 files, 188KB
Removing: /usr/local/Cellar/htop/2.2.0... (11 files, 188.0KB)
==> Upgrading speedtest-cli
==> Downloading https://github.com/sivel/speedtest-cli/archive/v2.1.1.tar.gz
==> Downloading from https://codeload.github.com/sivel/speedtest-cli/tar.gz/v2.1.1
######################################################################## 100.0%
🍺  /usr/local/Cellar/speedtest-cli/2.1.1: 7 files, 83.2KB, built in 5 seconds
Removing: /usr/local/Cellar/speedtest-cli/2.0.2... (7 files, 81.2KB)
==> Upgrading lastpass-cli
==> Installing dependencies for lastpass-cli: curl, libgpg-error, libassuan and pinentry
==> Installing lastpass-cli dependency: curl
==> Downloading https://homebrew.bintray.com/bottles/curl-7.64.0.mojave.bottle.tar.gz
######################################################################## 100.0%
==> Pouring curl-7.64.0.mojave.bottle.tar.gz
==> Caveats
curl is keg-only, which means it was not symlinked into /usr/local,
because macOS already provides this software and installing another version in
parallel can cause all kinds of trouble.

If you need to have curl first in your PATH run:
  echo 'export PATH="/usr/local/opt/curl/bin:$PATH"' >> ~/.bash_profile

For compilers to find curl you may need to set:
  export LDFLAGS="-L/usr/local/opt/curl/lib"
  export CPPFLAGS="-I/usr/local/opt/curl/include"

For pkg-config to find curl you may need to set:
  export PKG_CONFIG_PATH="/usr/local/opt/curl/lib/pkgconfig"


zsh completions have been installed to:
  /usr/local/opt/curl/share/zsh/site-functions
==> Summary
🍺  /usr/local/Cellar/curl/7.64.0: 449 files, 3.2MB
==> Installing lastpass-cli dependency: libgpg-error
==> Downloading https://homebrew.bintray.com/bottles/libgpg-error-1.36.mojave.bottle.tar.gz
######################################################################## 100.0%
==> Pouring libgpg-error-1.36.mojave.bottle.tar.gz
🍺  /usr/local/Cellar/libgpg-error/1.36: 26 files, 856KB
==> Installing lastpass-cli dependency: libassuan
==> Downloading https://homebrew.bintray.com/bottles/libassuan-2.5.3.mojave.bottle.tar.gz
######################################################################## 100.0%
==> Pouring libassuan-2.5.3.mojave.bottle.tar.gz
🍺  /usr/local/Cellar/libassuan/2.5.3: 16 files, 444.2KB
==> Installing lastpass-cli dependency: pinentry
==> Downloading https://homebrew.bintray.com/bottles/pinentry-1.1.0_1.mojave.bottle.tar.gz
######################################################################## 100.0%
==> Pouring pinentry-1.1.0_1.mojave.bottle.tar.gz
🍺  /usr/local/Cellar/pinentry/1.1.0_1: 12 files, 263.9KB
==> Installing lastpass-cli
==> Downloading https://homebrew.bintray.com/bottles/lastpass-cli-1.3.1_1.mojave.bottle.tar.gz
######################################################################## 100.0%
==> Pouring lastpass-cli-1.3.1_1.mojave.bottle.tar.gz
==> Caveats
Bash completion has been installed to:
  /usr/local/etc/bash_completion.d

zsh completions have been installed to:
  /usr/local/share/zsh/site-functions
==> Summary
🍺  /usr/local/Cellar/lastpass-cli/1.3.1_1: 10 files, 215.1KB
Removing: /usr/local/Cellar/lastpass-cli/1.2.2... (9 files, 205.5KB)
Removing: /usr/local/Cellar/lastpass-cli/1.3.1... (9 files, 208.3KB)
==> Upgrading inetutils
==> Downloading https://homebrew.bintray.com/bottles/inetutils-1.9.4_2.mojave.bottle.tar.gz
######################################################################## 100.0%
==> Pouring inetutils-1.9.4_2.mojave.bottle.tar.gz
==> Caveats
The following commands have been installed with the prefix 'g'.

    dnsdomainname
    ftp
    rcp
    rexec
    rlogin
    rsh
    telnet

If you really need to use these commands with their normal names, you
can add a "gnubin" directory to your PATH from your bashrc like:

    PATH="/usr/local/opt/inetutils/libexec/gnubin:$PATH"
==> Summary
🍺  /usr/local/Cellar/inetutils/1.9.4_2: 107 files, 2.6MB
Removing: /usr/local/Cellar/inetutils/1.9.4... (107 files, 2.6MB)
Removing: /usr/local/Cellar/inetutils/1.9.4_1... (107 files, 2.6MB)
==> Upgrading pcre
==> Downloading https://homebrew.bintray.com/bottles/pcre-8.43.mojave.bottle.tar.gz
######################################################################## 100.0%
==> Pouring pcre-8.43.mojave.bottle.tar.gz
🍺  /usr/local/Cellar/pcre/8.43: 204 files, 5.5MB
Removing: /usr/local/Cellar/pcre/8.42... (204 files, 5.3MB)
==> Upgrading winetricks
==> Downloading https://github.com/Winetricks/winetricks/archive/20190310.tar.gz
==> Downloading from https://codeload.github.com/Winetricks/winetricks/tar.gz/20190310
######################################################################## 100.0%
Error: The `brew link` step did not complete successfully
The formula built, but is not symlinked into /usr/local
Could not symlink bin/winetricks
Target /usr/local/bin/winetricks
already exists. You may want to remove it:
  rm '/usr/local/bin/winetricks'

To force the link and overwrite all conflicting files:
  brew link --overwrite winetricks

To list all files that would be deleted:
  brew link --overwrite --dry-run winetricks

Possible conflicting files are:
/usr/local/bin/winetricks
==> Caveats
winetricks is a set of utilities for wine, which is installed separately:
  brew install wine
==> Summary
🍺  /usr/local/Cellar/winetricks/20190310: 6 files, 801.4KB, built in 6 seconds
Removing: /usr/local/Cellar/winetricks/20181203... (6 files, 787.3KB)
==> Upgrading aircrack-ng
==> Downloading https://homebrew.bintray.com/bottles/aircrack-ng-1.5.2.mojave.bottle.tar.gz
######################################################################## 100.0%
==> Pouring aircrack-ng-1.5.2.mojave.bottle.tar.gz
==> Caveats
Run `airodump-ng-oui-update` install or update the Airodump-ng OUI file.
==> Summary
🍺  /usr/local/Cellar/aircrack-ng/1.5.2: 60 files, 1.7MB
Removing: /usr/local/Cellar/aircrack-ng/1.1_2... (35 files, 931KB)
Removing: /usr/local/Cellar/aircrack-ng/1.3... (60 files, 1.7MB)
Removing: /usr/local/Cellar/aircrack-ng/1.4... (60 files, 1.7MB)
==> Upgrading ncdu
==> Downloading https://homebrew.bintray.com/bottles/ncdu-1.14.mojave.bottle.tar.gz
######################################################################## 100.0%
==> Pouring ncdu-1.14.mojave.bottle.tar.gz
🍺  /usr/local/Cellar/ncdu/1.14: 7 files, 95.5KB
Removing: /usr/local/Cellar/ncdu/1.12... (7 files, 87.6KB)
Removing: /usr/local/Cellar/ncdu/1.13... (7 files, 89.6KB)
==> Upgrading pyenv
==> Downloading https://homebrew.bintray.com/bottles/pyenv-1.2.9.mojave.bottle.tar.gz
######################################################################## 100.0%
==> Pouring pyenv-1.2.9.mojave.bottle.tar.gz
🍺  /usr/local/Cellar/pyenv/1.2.9: 619 files, 2.4MB
Removing: /usr/local/Cellar/pyenv/1.1.2... (581 files, 2.2MB)
Removing: /usr/local/Cellar/pyenv/1.1.3... (581 files, 2.2MB)
Removing: /usr/local/Cellar/pyenv/1.1.5... (594 files, 2.3MB)
Removing: /usr/local/Cellar/pyenv/1.2.1... (584 files, 2.3MB)
Removing: /usr/local/Cellar/pyenv/1.2.5... (606 files, 2.4MB)
Removing: /usr/local/Cellar/pyenv/1.2.8... (612 files, 2.4MB)
==> Caveats
==> openssl@1.1
A CA file has been bootstrapped using certificates from the system
keychain. To add additional certificates, place .pem files in
  /usr/local/etc/openssl@1.1/certs

and run
  /usr/local/opt/openssl@1.1/bin/c_rehash

openssl@1.1 is keg-only, which means it was not symlinked into /usr/local,
because openssl/libressl is provided by macOS so don't link an incompatible version.

If you need to have openssl@1.1 first in your PATH run:
  echo 'export PATH="/usr/local/opt/openssl@1.1/bin:$PATH"' >> ~/.bash_profile

For compilers to find openssl@1.1 you may need to set:
  export LDFLAGS="-L/usr/local/opt/openssl@1.1/lib"
  export CPPFLAGS="-I/usr/local/opt/openssl@1.1/include"

For pkg-config to find openssl@1.1 you may need to set:
  export PKG_CONFIG_PATH="/usr/local/opt/openssl@1.1/lib/pkgconfig"

==> sqlite
sqlite is keg-only, which means it was not symlinked into /usr/local,
because macOS provides an older sqlite3.

If you need to have sqlite first in your PATH run:
  echo 'export PATH="/usr/local/opt/sqlite/bin:$PATH"' >> ~/.bash_profile

For compilers to find sqlite you may need to set:
  export LDFLAGS="-L/usr/local/opt/sqlite/lib"
  export CPPFLAGS="-I/usr/local/opt/sqlite/include"

For pkg-config to find sqlite you may need to set:
  export PKG_CONFIG_PATH="/usr/local/opt/sqlite/lib/pkgconfig"

==> python
Python has been installed as
  /usr/local/bin/python3

Unversioned symlinks `python`, `python-config`, `pip` etc. pointing to
`python3`, `python3-config`, `pip3` etc., respectively, have been installed into
  /usr/local/opt/python/libexec/bin

If you need Homebrew's Python 2.7 run
  brew install python@2

You can install Python packages with
  pip3 install <package>
They will install into the site-package directory
  /usr/local/lib/python3.7/site-packages

See: https://docs.brew.sh/Homebrew-and-Python
==> azure-cli
Bash completion has been installed to:
  /usr/local/etc/bash_completion.d
==> cmake
Emacs Lisp files have been installed to:
  /usr/local/share/emacs/site-lisp/cmake
==> wine
You may also want winetricks:
  brew install winetricks
==> ncurses
ncurses is keg-only, which means it was not symlinked into /usr/local,
because macOS already provides this software and installing another version in
parallel can cause all kinds of trouble.

If you need to have ncurses first in your PATH run:
  echo 'export PATH="/usr/local/opt/ncurses/bin:$PATH"' >> ~/.bash_profile

For compilers to find ncurses you may need to set:
  export LDFLAGS="-L/usr/local/opt/ncurses/lib"
  export CPPFLAGS="-I/usr/local/opt/ncurses/include"

For pkg-config to find ncurses you may need to set:
  export PKG_CONFIG_PATH="/usr/local/opt/ncurses/lib/pkgconfig"

==> htop
htop requires root privileges to correctly display all running processes,
so you will need to run `sudo htop`.
You should be certain that you trust any software you grant root privileges.
==> curl
curl is keg-only, which means it was not symlinked into /usr/local,
because macOS already provides this software and installing another version in
parallel can cause all kinds of trouble.

If you need to have curl first in your PATH run:
  echo 'export PATH="/usr/local/opt/curl/bin:$PATH"' >> ~/.bash_profile

For compilers to find curl you may need to set:
  export LDFLAGS="-L/usr/local/opt/curl/lib"
  export CPPFLAGS="-I/usr/local/opt/curl/include"

For pkg-config to find curl you may need to set:
  export PKG_CONFIG_PATH="/usr/local/opt/curl/lib/pkgconfig"


zsh completions have been installed to:
  /usr/local/opt/curl/share/zsh/site-functions
==> lastpass-cli
Bash completion has been installed to:
  /usr/local/etc/bash_completion.d

zsh completions have been installed to:
  /usr/local/share/zsh/site-functions
==> inetutils
The following commands have been installed with the prefix 'g'.

    dnsdomainname
    ftp
    rcp
    rexec
    rlogin
    rsh
    telnet

If you really need to use these commands with their normal names, you
can add a "gnubin" directory to your PATH from your bashrc like:

    PATH="/usr/local/opt/inetutils/libexec/gnubin:$PATH"
==> winetricks
winetricks is a set of utilities for wine, which is installed separately:
  brew install wine
==> aircrack-ng
`

`flo$ winecfg
dyld: Library not loaded: /usr/local/Cellar/wine/4.0/lib/libwine.1.dylib
  Referenced from: /usr/local/Cellar/wine/4.0/lib/../bin/wineserver
  Reason: no suitable image found.  Did find:
	/usr/local/Cellar/wine/4.0/lib/libwine.1.dylib: mach-o, but wrong architecture
	/usr/local/Cellar/wine/4.0/lib/libwine.1.dylib: stat() failed with errno=1
	/usr/local/lib/libwine.1.dylib: mach-o, but wrong architecture
	/usr/local/Cellar/wine/4.0/lib/libwine.1.0.dylib: mach-o, but wrong architecture
	/usr/local/Cellar/wine/4.0/lib/libwine.1.0.dylib: mach-o, but wrong architecture`

**Comment by flotpg** [comment]
Switching to 3.0.2 worked for me.
On my macOS Mojave install it is /usr/local/**C**ellar/ ...

**Comment by flotpg** [comment]
I already tried uninstalling brew completely (did a brew dump before). Still the same issue.
On my other mac it works fine with wine 4.0

**Comment by Blatman** [comment]
FWIW - I tried Wine 4.4 Devel and it failed and had to go back to Staging 3.15 (Staging 3.16 also failed). Looking at a few forums,etc there was a suggestion that 32/64bit files were maybe a bit mixed up in the Wine install so I renamed the .wine folder and tried 4.4 again - it worked OK with rcedit but failed with winresourcer. The reason for the winresourcer fail was that the new install was set for Win7 and when set back to Win XP all was fine again. BTW this was not using the homebrew install but just using the standard MacOS .app install packages. At some stage I had probably specified the 64bit option on the Wine install but I think that option only relates to running 64bit apps and I guess both rcedit/winresourcer are 32bit although I notice that a x64 version of rcedit is available on github.
